.idea/codeStyles/Project.xml

```diff
@@ -1,8 +1,120 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <AndroidXmlCodeStyleSettings>
```

Comment:
nit: let's gitignore this file :)

Reply:
So I found out that is not best practices to gitignore this file though I found that we were not ignoring some files that we should have been

Reply:
hm! I guess it's to keep the Android dev settings in common for developers